data/Localization/german_(germany)/global.ini

```diff
@@ -58764,7 +58764,7 @@ kareah_datapad_repeat_02_msg=Gemäß unseren verschärften Sicherheitsprotokolle
 kareah_datapad_repeat_02_recipient=An: Autorisiertes Personal
 kareah_datapad_repeat_02_subject=Thema: Automatisches Zurücksetzen der Sicherheit
 kareahsweep_Kill_Counter_UI=Verbleibende Feinde %ls
-kareahsweep_combatpay_0001=Zusätzlich zur normalen Vertragszahlung gibt es einen Bonus für jeden erfolgreich neutralisierten Kämpfer.
+kareahsweep_combatpay_0001=Zusätzlich zur normalen Auftragszahlung gibt es einen Bonus für jeden erfolgreich neutralisierten Kämpfer.
 kareahsweep_combatpay_0002=Leider konnten wir nicht feststellen, wie groß die Truppe ist, mit der ihr es zu tun habt, aber ihr erhaltet eine Gefahrenzulage für jeden Feind, den ihr erfolgreich besiegt habt.
 kareahsweep_combatpay_0003=Ich habe auch die Erlaubnis erhalten, für jeden Geächteten, um den ihr euch kümmert, eine variable Gefahrenzulage zu zahlen.
 kareahsweep_combatpay_0004=Und um Ihre Frage vorsorglich zu beantworten: Ja, zusätzlich zu der normalen Pauschalgebühr wird für jedes Hostile, das Sie von der Website löschen, eine Hazard Pay-Gebühr erhoben.
```

Comment:
Änderung in "
"Um Ihre Frage vorwegzunehmen: Ja, zusätzlich zu der normalen Pauschalgebühr wird eine Gefahrenzulage für jeden Feind ausgegeben, den Sie von dem Standort entfernen." " empfohlen